city/crud.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from . import models, schemas
+
+
+async def create_city(db: AsyncSession, city: schemas.CityCreate):
+    db_city = models.City(**city.dict())
```

Comment:
Verify that `models.City` is correctly defined in the `models` module. This is necessary for creating a new city record in the database.